Containerfile

```diff
@@ -1,45 +1,55 @@
-FROM registry.fedoraproject.org/fedora-minimal:39
-LABEL maintainer="Red Hat"
+# hadolint global ignore=DL3007
```

Comment:
Why do you want to use latest? I'd prefer dependabot to update versions and have reproducible builds in case we need to revert

Reply:
Because this runs on the latest upstream head. With a static image digest we won't be quick enough to react to potentially breaking changes (e.g. testing early with a Fedora Rawhide images to prepare for coming changes in GCC or whatnot, not for gating though!), especially with purposefully decreasing dependabot's cadence of updates to "monthly". Reproducible builds are IMO only part of the testing strategy and part of the whole picture, on one hand you want stable tests in a predictable environment to limit all your potential errors coming from the project itself; on the other hand though, you also want early failures coming from the outside environment (and IMO you want platform ones rather quickly). Not having an image digest is not that much different from actually having one and let dependabot spam updates about it. I'm ambivalent on this, though I'm not fond of dependabot taking care of this since the digest is completely opaque. Does dependabot pair container image digest updates with something like changelog changes, IOW package updates? Because I don't see changelogs being available on docker hub itself and if that's the case then having dependabot opening a PR for a digest update doesn't really improve the situation with using 'latest' straight away.

Reply:
With latest you will have breaking changes more often noticing them at very last phase.

With pinning and dependabot you can see what changed and have running CI to avoid merging breaking change and notice them just before relase.

Frankly monthly dependabot updares has low cadence to address CVE in modules. But that's unrelated.

> Not having an image digest is not that much different from actually having one and let dependabot spam updates about it.
It is quite a difference, to have image broken without knowing it and have stable image and see when a update is breaking things in the PR opened by dependabot.

> you also want early failures coming from the outside environment (and IMO you want platform ones rather quickly)
You will have early failures, just in dependabot PR without affecting work in progress work and other PRs randomly.

> digest update doesn't really improve the situation with using 'latest' straight away
I completely disagree, you have no control what's in latest, sometime latest images are even dev versions